3.1 - Add filter options and links (#326)
akeneo/pim-helpcenter: content/md/products-super-power/enrich-your-reference-entity-records.md

> - Finally, the grid can display up to 500 results using an **infinite scroll**. If you have more results, please filter your search. When reaching this limit, the grid displays the message *"You just scrolled through the first 500 results out of 1450 records. To see more, try with more specific search criteria."*

Reviewer: ```suggestion
- Finally, the grid comes with an **infinite scroll** that allows you to display up to 500 results. If you reach this limit, the grid will show the following message *"You just scrolled through the first 500 results out of 1450 records. To see more, try with more specific search criteria."*
```

::: info
In the records grid in 3.0, you will see only certain types of attributes: code, text and image.  

> _Since 3.1_, the values of the attributes types single/multiple options and reference entity single/multiple links are also displayed in the records grid.

Reviewer: ```suggestion
_Since the 3.1 version_, the values of the attributes types single/multiple options and reference entity single/multiple links can also be displayed in the records grid.
```

:::

::: info

> The order of the columns in the records grid is "Image, Label, Code, Complete", the other attributes are then ordered according to their order of creation in the "Attributes" tab.

Reviewer: The columns' order in the records grid is frozen, it cannot be changed. By default, the grid's column order is `Image`, `Label`, `Code`, `Complete`. If you add other attributes, under the `Attributes` tab, the grid will display them according to their order of creation.

To know how to define the required attributes for the completeness, please take a look at this [article](/articles/manage-reference-entities.html)
:::

## Filter on options and reference entity links (_since 3.1_)

> _Since 3.1_, we added the ability to filter on attributes types single/multiple options or reference entity single/multiple links in the records grid. It allows an exact search to find the records with a specific value (option value or link to a record).

Reviewer: ```suggestion
_Since the 3.1 version_, we added the possibility to filter on single/multiple options or reference entity single/multiple links attributes types in the records grid. This filter allows an exact search on an reference entity option value or on a reference entity link. 
```

> For examples:

Reviewer 2: Je pense qu'il n'y a pas de "s" quand on met "for" avant.

Reviewer: you can use "for instance", or "e.g."

> For examples:

Reviewer: ```suggestion
For instance:
```

- If a "Brand" reference entity has a "Designers" attribute linked to a "Designer" reference entity, you can search for all brands with the designer "Ron Arad".
- If a "Brand" reference entity has a "Country" attribute with a list of options, you can search for brands with the country "Italy".

![Records grid](../img/Records_filter_options.png)

> The operator is an "in list" operator. If you are looking for the brand with the designer "Philippe Starck" OR "Ron Arad", the brands with the designer "Philippe Starck" OR the designer "Ron Arad" will be displayed.

Reviewer 3: ```suggestion
The operator is an `in list` operator. If you are looking for the brand with the designer "Philippe Starck" OR "Ron Arad", the brands with the designer "Philippe Starck" OR the designer "Ron Arad" will be displayed.
```

> You can combine the filters. For example, you can search for all brands with the country "Italy" AND the designer "Ron Arad".

Reviewer: ```suggestion
You can combine filters. For instance, you can search on all brands with the country "Italy" AND the designer "Ron Arad".
```

Reviewer: this example is not very clear to me :(
